src/ts/dom.ts

```diff
@@ -25,4 +36,31 @@ export class Dom {
     document.body.style.overflow = 'hidden';
   }
 
+  static getInputValue(a: string): string {
+    return (<HTMLInputElement>this.findById(a)).value;
+  }
+
+  static clearInputValue(a: string): void {
+    (<HTMLInputElement>this.findById(a)).value = '';
+  }
+
+  static focusOn(a: string): void {
+    this.findById(a).focus();
+  }
+
+  static addHTMLElement(parentElement: HTMLElement, childElementType: string, childElementOptions?: ChildElementOptions): HTMLElement | HTMLInputElement | HTMLSelectElement | HTMLTextAreaElement | HTMLProgressElement | HTMLCanvasElement {
+    var newElement = document.createElement(childElementType);
+    if(childElementOptions !== undefined) {
+      Object.keys(childElementOptions).forEach(key => {
```

Comment:
is it a better way to loop through object properties ?
the usual : 
for (let p in myObject) {
    if (myObject.hasOwnProperty(p)) {

Reply:
Exact same result with Object.keys (https://leftshift.io/4-javascript-optimisations-you-should-know - part 2), but differences on browser compatibility: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys
Let me know your choice :)

Reply:
OK let's keep it that way.
Thanks for the infos.